plugins/tasks/lock/src/main/java/com/walmartlabs/concord/plugins/lock/LockTaskCommon.java

```diff
@@ -82,8 +82,8 @@ private String checkScope(String scope) {
             return null;
         }
 
-        if (!scope.equalsIgnoreCase("ORG")
-                && !scope.equalsIgnoreCase("PROJECT")) {
+        if (!"ORG".equalsIgnoreCase(scope)
```

Comment:
Checked before.

Reply:
The purpose of this change is to be proactively defensive. Though it may be negligibly faster to run this way, it also conforms to a best practice and makes the code easier to read. Also, since the behavior is otherwise equivalent, there is no cost to this change.